google/cloud/bigquerycontrol/README.md

```diff
@@ -54,14 +58,11 @@ int main(int argc, char* argv[]) try {
 
 ## More Information
 
-- Official documentation about the \[BigQuery API\]\[cloud-service-docs\]
-  service
-- \[Reference doxygen documentation\]\[doxygen-link\] for each release of this
+- Official documentation about the [BigQuery API][cloud-service-docs] service
+- [Reference doxygen documentation][doxygen-link] for each release of this
   client library
-- Detailed header comments in our \[public `.h`\]\[source-link\] files
+- Detailed header comments in our [public `.h`][source-link] files
 
-\[cloud-service-docs\]: https://cloud.google.com/bigquerycontrol \[EDIT HERE\]
```

Comment:
comment: Weird. I guess when a service doesn't have that product documentation field in the yaml, we get an `[EDIT HERE]` marker which confuses `checkers`.